app/models/region.rb

```diff
@@ -88,10 +88,10 @@ def syncable_patients
     case region_type
       when "block"
         registered_patients.with_discarded
-          .union(assigned_patients.with_discarded)
+          .or(assigned_patients.with_discarded)
           .union(appointed_patients.with_discarded)
```

Comment:
Why does this remain a `union`?

Reply:
Can't `or` with a scope that has `includes` or `joins`.